props/blaster.h

```diff
@@ -87,6 +87,8 @@ mdauto		Sound made when switching to AUTO mode
 #ifndef PROPS_BLASTER_H
 #define PROPS_BLASTER_H
 
+#define PROP_HAS_GETBLASTERMODE
```

Comment:
I don't remember what this was for. OLED maybe? Also not sure why I have it in my local copy but it's not in the master.

Reply:
It's used here:
https://github.com/profezzorn/ProffieOS/blob/b88b82da47b2c5b02691bc216c150edf2613a53b/ProffieOS.ino#L695
which is in turn used by BlasterModeF

Reply:
ah yes I remember that. Cool. I guess it just never made it into the blaster prop?